### ISSUE TITLE
Bug: exit status 127 after updating to the latest tag

### ISSUE THREAD

**Issue by avt00** [qdm12/gluetun#1420]
### Is this urgent?

Yes

### Host OS

Synology

### CPU arch

x86_64

### VPN service provider

Custom

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2023-02-26T19:18:36.074Z (commit ac5141b)

### What's the problem 🤔

After Watchtower has pulled the latest image from Docker Hub, the Gluetun container can't start.

### Share your logs

```plain text
gluetun    | Running version latest built on 2023-02-26T19:18:36.074Z (commit ac5141b)
gluetun    | 
gluetun    | 🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
gluetun    | 🐛 Bug? https://github.com/qdm12/gluetun/issues/new
gluetun    | ✨ New feature? https://github.com/qdm12/gluetun/issues/new
gluetun    | ☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
gluetun    | 💻 Email? quentin.mcgaw@gmail.com
gluetun    | 💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
gluetun    | 2023-02-26T23:13:22+03:00 INFO [routing] default route found: interface eth0, gateway 172.22.0.1 and assigned IP 172.22.0.3
gluetun    | 2023-02-26T23:13:22+03:00 INFO [routing] local ethernet link found: eth0
gluetun    | 2023-02-26T23:13:22+03:00 INFO [routing] local ipnet found: 172.22.0.0/16
gluetun    | 2023-02-26T23:13:22+03:00 INFO [firewall] enabling...
gluetun    | 2023-02-26T23:13:22+03:00 INFO [firewall] enabled successfully
gluetun    | 2023-02-26T23:13:23+03:00 INFO [storage] merging by most recent 13163 hardcoded servers and 13163 servers read from /gluetun/servers.json
gluetun    | 2023-02-26T23:13:23+03:00 INFO Alpine version: 3.17.2
gluetun    | 2023-02-26T23:13:24+03:00 ERROR exit status 127
gluetun    | 2023-02-26T23:13:24+03:00 INFO Shutdown successful
```

### Share your configuration

```yml
version: "3"
services:
  gluetun:
    image: qmcgaw/gluetun:v3.32.0
    networks:
      gluetun_network:
    container_name: gluetun
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    volumes:
      - /volume1/docker/gluetun:/gluetun
    ports:
      - 8888:8888/tcp #
      - 9117:9117 #
      - 8989:8989 #
    environment:
      - FIREWALL_OUTBOUND_SUBNETS=172.22.0.0/16
      - VPN_SERVICE_PROVIDER=custom
      - VPN_TYPE=wireguard
      - VPN_ENDPOINT_IP=
      - VPN_ENDPOINT_PORT=
      - WIREGUARD_PUBLIC_KEY=
      - WIREGUARD_PRIVATE_KEY=
      - WIREGUARD_ADDRESSES=
      - HTTPPROXY=on
      - HTTPPROXY_LOG=on
networks:
  gluetun_network:
    external: true
```

**Comment by drumtechphoto** [comment]
Having the same issue.  Watchtower automatically polls hourly for all container updates.   I don't see any new releases posted?  What could have caused this issue?   Its like a restart loop...

**Comment by stenehall** [comment]
> 

https://hub.docker.com/r/qmcgaw/gluetun/tags last releaste 2 hours ago if you're running `:latest`

**Comment by stenehall** [comment]
> Having the same issue. Watchtower automatically polls hourly for all container updates. I don't see any new releases posted? What could have caused this issue? Its like a restart loop...

https://hub.docker.com/r/qmcgaw/gluetun/tags last releaste 2 hours ago if you're running `:latest`

**Comment by drumtechphoto** [comment]
> > 
> 
> https://hub.docker.com/r/qmcgaw/gluetun/tags last releaste 2 hours ago if you're running `:latest`

Thank you for sharing.  Not sure why its not posted under the releases here yet?

**Comment by EthanC** [comment]
+1  with NordVPN using OpenVPN authentication.

```
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
Running version latest built on 2023-02-26T19:18:36.074Z (commit ac5141b)
🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-02-26T15:23:52-06:00 INFO [routing] default route found: interface eth0, gateway 172.24.0.1 and assigned IP 172.24.0.4
2023-02-26T15:23:52-06:00 INFO [routing] local ethernet link found: eth0
2023-02-26T15:23:52-06:00 INFO [routing] local ipnet found: 172.24.0.0/16
2023-02-26T15:23:52-06:00 INFO [firewall] enabling...
2023-02-26T15:23:52-06:00 INFO [firewall] enabled successfully
2023-02-26T15:23:52-06:00 INFO [storage] merging by most recent 13163 hardcoded servers and 13163 servers read from /gluetun/servers.json
2023-02-26T15:23:52-06:00 INFO Alpine version: 3.17.2
2023-02-26T15:23:52-06:00 ERROR exit status 127
2023-02-26T15:23:52-06:00 INFO Shutdown successful
```

**Comment by dedworks** [comment]
> Thank you for sharing. Not sure why its not posted under > https://github.com/qdm12/gluetun/releases

That's because it's not a stable release.

> Finally you can also **use image tag :v3 to get the latest stable release** (for example v3.28.2), although I would appreciate if you can use the latest image instead to detect user issues faster!

https://github.com/qdm12/gluetun/wiki/Docker-image-tags

My tunnels are back up for now on v3.32.0 after switching to `image: qmcgaw/gluetun:v3` and letting `docker-compose up` re-create the container.

**Comment by drumtechphoto** [comment]
> > **e image tag :v3 to get the latest stable release** (for example v3.28.2), although I would appreciate if you can use the latest image instead to detect user issues faster!

Thanks for the reply.  With the help of using the image: qmcgaw/gluetun:v3 > in my portainer stack, I was able to recreate the container and it started back up as nomal.

**Comment by sinjinsmythe** [comment]
Thanks for sharing this fix, I've also revered to v3 and working again, tearing my hear out all night!

**Comment by JFPCreations** [comment]
> > > **e image tag :v3 to get the latest stable release** (for example v3.28.2), although I would appreciate if you can use the latest image instead to detect user issues faster!
> 
> Thanks for the reply. With the help of using the image: qmcgaw/gluetun:v3 > in my portainer stack, I was able to recreate the container and it started back up as nomal.

Thank you this helped me.

**Comment by gionag** [comment]
same here
`exit status: 127`

**Comment by royaltongue** [comment]
Can confirm running `image: qmcgaw/gluetun:v3` fixes this issue

**Comment by qdm12** [comment]
Fixed in the latest image, see #1421
